### PR TITLE
More visibility to SmallRye

### PR DESCRIPTION
**Pull request by radcortez**

### CODE REVIEW

**Review by gastaldi — APPROVED** [review]
LGTM

**Review by gsmet — DISMISSED** [review]
What's the rationale of this? I read a lot of things about people being happy to have Quarkus compatible with MicroProfile. Will it be visible enough after this change?

Also, you will need to install redirects in the website.

File: docs/src/main/asciidoc/config-reference.adoc [hunk 1]
```diff
@@ -13,10 +13,9 @@ include::./attributes.adoc[]
 :toc:
 
 In this reference guide we're going to describe various aspects of Quarkus configuration.
-A Quarkus application and Quarkus itself (core and extensions) are both configured via the same mechanism that leverages the https://microprofile.io/project/eclipse/microprofile-config[MicroProfile Config] API.
-Quarkus configuration is based on https://github.com/smallrye/smallrye-config[SmallRye Config], an implementation of the MicroProfile Config specification.
-All of the standard features are supported.
-Moreover, there are several additional features which are made available by the SmallRye Config project as well as by Quarkus itself.
+A Quarkus application and Quarkus itself (core and extensions) are both configured via the same mechanism that leverages
+the https://github.com/smallrye/smallrye-config[SmallRye Config] API an implementation of the MicroProfile Config
+specification. Moreover, there are several additional features which are made available by Quarkus itself.
```
Comment (kenfinnigan) [inline docs/src/main/asciidoc/config-reference.adoc:18]: Would be good to link to the MP Config spec like was done before

File: docs/src/main/asciidoc/micrometer.adoc [hunk 2]
```diff
@@ -402,9 +402,9 @@ to inject any configuration attributes you need to configure the registry. Most
 like `quarkus-micrometer-registry-statsd`, define a producer for registry-specific configuration objects
 that are integrated with the Quarkus configuration model.
 
-== Support for the MicroProfile Metrics API
```
Comment (kenfinnigan) [inline docs/src/main/asciidoc/micrometer.adoc:405]: These changes should be reverted.

This section is specifically about enabling the use of MP Metrics APIs with Micrometer, nothing to do with SmallRye Metrics.

cc @ebullient

**Review by kenfinnigan — CHANGES_REQUESTED** [review]

**Comment by kenfinnigan** [comment]
@gsmet There was a discussion yesterday around making it clearer that Quarkus is using SmallRye projects, which does implement MicroProfile specifications, to make it easier to highlight when SmallRye goes beyond a specification

**Comment by gsmet** [comment]
There are now conflicts due to the master -> main change.

**Comment by radcortez** [comment]
> There are now conflicts due to the master -> main change.

Fixed.

Also added the redirects in https://github.com/quarkusio/quarkusio.github.io/pull/925. Do we need to add an initial version of the renamed guides or everything is synced automatically?

**Comment by gsmet** [comment]
@radcortez no, they will be added automatically. I just need to think about merging the PR after the release :).

@kenfinnigan could you have another look?

**Review by kenfinnigan — APPROVED** [review]

**Comment by radcortez** [comment]
Thanks!

**Pull request merged**